gap/base/recognition.gi

```diff
@@ -970,9 +970,37 @@ InstallGlobalFunction( "GetCompositionTreeNode",
 
 # Testing:
 
-RECOG.TestGroup := function(g,proj,size)
-  local l,r,ri,s,x,count,lvl,seedMT,seedRS,gens;
+RECOG.TestGroupOptions := rec(
+      # Number of times to test size of RecogniseGroup is right
+      sizeTests := 3,
+
+      # Number of random elements in group to check
+      # This is used both for the number of elements in the
+      # group to check, and the number of random elements of
+      # a supergroup to check
+      inTests := 30,
+
+      # The following options are off by default as they fail on
+      # many examples at present
+      tryNonGroupElements := false
```

Comment:
Shouldn't there be an empty line between the preceeding comment and this, and then perhaps a new comment inserted which explains what `tryNonGroupElements` is about? Like this:
```suggestion

      # if the following is set to true, then we test what happens if  SLPforElement
      # is called with elements outside the group
      tryNonGroupElements := false
```